### PR TITLE
API backlog WG - API backlog call MoM -  2nd Nov 2023

### PR DESCRIPTION
**Pull request by jordonezlucena**
API backlog WG - API backlog call MoM -  2nd Nov 2023

### CODE REVIEW

File: APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md
Comment (hdamker) [inline APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md:61]: The proposal is now in #333, as mentioned in https://github.com/camaraproject/WorkingGroups/issues/317.

Comment (jordonezlucena) [reply APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md]: Good catch. Fixed

File: APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md
Comment (hdamker) [inline APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md:60]: Not explicitly discussed within TSC, but as there was no objection, I asked @MarkusKuemmerle to create the new repository as requested within this [message](https://lists.camaraproject.org/g/tsc/message/62).

File: APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md
Comment (hdamker) [inline APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md:59]: You might want to add a note that the TSC on Nov 2nd approved the creation of the sub project. The minutes are still in work, but please go forward to provide the maintainers.

File: APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md
Comment (jordonezlucena) [reply APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md:60]: I will wait for the official confirmation from TSC to close this action point, for full transparency

File: APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md
Comment (jordonezlucena) [reply APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md]: Done. Thank you!

**Review by hdamker — DISMISSED** [review]
My comments are addressed, thanks.

**Review by eric-murray — APPROVED** [review]

**Comment by eric-murray** [comment]
@jordonezlucena
I clarified my comments on KYC Age Verification, which is that Vodafone support including this in the existing KnowYourCustomer sub-project. We will not formally support this specific API proposal as such, so I removed that action.

**Review by MarkCornall — APPROVED** [review]
no additional comments

**Pull request merged**
